src/platform/elements/form/extras/address/Address.ts

```diff
@@ -287,7 +292,8 @@ export class NovoAddressElement implements ControlValueAccessor, OnInit {
   }
 
   updateStates() {
-    if (this.config.state.pickerConfig.options && !Helpers.isBlank(this.model.countryID)) {
+    if (this.config.state.pickerConfig.options && !Helpers.isBlank(this.model.countryID) &&
+      !(this.config.readOnly || this.config.state.readOnly)) {
```

Comment:
if we don't preload states when state is disabled, it will not automatically preload when state picker is enabled. this disabled check here isn't needed.

Reply:
Fixed.

Reply:
i don't see the update